### ISSUE TITLE
[BUG] Arguments input with a "-" sign do not get parsed.

### ISSUE THREAD

**Issue by hackabean** [byt3bl33d3r/SILENTTRINITY#109]
When setting up an argument (or any value) for the command that requires "-" sign the argument does not get parsed by ST. It affects all modules not only the one in the example.

for eg.
set Arguments "-c calc"
It is rather important as if you want to run bypassUAC modules through powershell you need to specify "-ep bypass" switch.

![image](https://user-images.githubusercontent.com/50614334/65816814-f7a91f00-e1ef-11e9-9fbc-ccf708e38eea.png)

**Comment by hackabean** [comment]
If somebody wants to use "-" sign as part of their command or arguments it needs a space to be parsed, something like this: " -ep bypass" , then it works.

**Comment by byt3bl33d3r** [comment]
I ran into this myself the other day, really weird not sure what could be causing this tbh.

**Comment by hackabean** [comment]
@byt3bl33d3r 
Seems like hyphen character removes either quotes or double quotes when used. That seems logical as "keyword arguments must be valid Python identifiers; these don't allow for - as that's reserved for subtraction." Sure enough it happens when setting Arguments "-h", shows help:

![image](https://user-images.githubusercontent.com/50614334/65964668-f4f73580-e44c-11e9-9d02-123c31bc44d1.png)


ref: https://stackoverflow.com/questions/24121279/python-keyword-arguments-with-hyphen
I have also found a post of a guy who had a similar issue:
https://metallapan.se/post/Sending-non-valid-names-as-arguments/

I have tried to see which part of the code was responsible for arguments parsing but I got lost. Perhaps will come useful to you.
